modules/ROOT/pages/_partials/endpoint-as2-send.adoc

```diff
@@ -0,0 +1,227 @@
+== AS2 Send Settings
+
+AS2 Send Settings pertain to trading partner-specific AS2 settings such as *certificate* and *URL*.
+
+[%header,cols="3s,7a"]
+|===
+|Setting |Description
+
+| URL
+
+|The HTTP/S address at which your partner is accepting incoming AS2 messages. +
+
+
+Required?::
+Yes, if AS2 is the transport protocol
+
+Valid values::
+
+Alphanumeric
+
+Default::
+
+None
+
+Example::
+`www.midsizesupplier.com:8080`
+
+| Subject
+
+|The field that specifies a default subject for messages you send out to this partner. +
+
+Required?::
+Yes
+
+Valid values::
+
+Alphanumeric
+
+Default::
+`None`
+
+| Encoding
+
+| Type of encoding to use when sending a message with AS2. +
+
+Required::
+Yes
+
+Default::
+Binary
+
+Valid values::
+
+* *Binary* +
+A method of encoding in which a message is represented using a combination of 0s and 1s.
+
+* *7-bit*
++
+Data that is all represented as relatively short lines with 998 bytes or less between CRLF line separation sequences.
+No bytes with decimal values greater than 127 are allowed and neither are NULLs (bytes with decimal value 0).
+CR (decimal value 13) and LF (decimal value 10) bytes only occur as part of CRLF line separation sequences.
+
+* *8-bit*
++
+Data that is all represented as relatively
```

Comment:
Delete "all" in each of these sections. What do we mean "represented"? Isn't this the actual data being sent? Or are we just referring to the encoding? If so, maybe say encoded instead of represented?

Reply:
Good question. Need to clarify.